pyrometheus/__init__.py

```diff
@@ -36,20 +36,24 @@
 
 # {{{ code generation helpers
 
+
 class CodeGenerationMapper(StringifyMapper):
     def map_constant(self, expr, enclosing_prec):
         return repr(expr)
 
     def map_if(self, expr, enclosing_prec, *args, **kwargs):
         return "np.where(%s, %s, %s)" % (
-                    self.rec(expr.condition, PREC_NONE, *args, **kwargs),
-                    self.rec(expr.then, PREC_NONE, *args, **kwargs),
-                    self.rec(expr.else_, PREC_NONE, *args, **kwargs))
+            self.rec(expr.condition, PREC_NONE, *args, **kwargs),
+            self.rec(expr.then, PREC_NONE, *args, **kwargs),
+            self.rec(expr.else_, PREC_NONE, *args, **kwargs),
+        )
```

Comment:
It looks like you may have run a code formatter, maybe "black"? If so, one reason that's not ideal is that it makes the diff hard to read, suggesting changes where there aren't any.

IMO, the correct approach to keeping Flake8 happy is to install an editor plugin that highlights problems, not to hope that some reformatter will fix everything.

Reply:
I'm using elpy in emacs, as recommended by Mike C. I did run black a couple of times when I just couldn't figure out what was wrong with the format (apparently some white spaces in line breaks). I naïvely did not anticipate it was going to touch other parts of the code. I'll avoid it going forward.